node/resolvers/search/index.ts

```diff
@@ -654,19 +659,24 @@ export const queries = {
 
     const [count, page] = getProductsCountAndPage(from, to)
 
-    const biggyArgs = {
+    const workspaceSearchParams = await getWorkspaceSearchParamsFromStorage(ctx)
```

Comment:
Don't you need to get the workspaceSearchParams on the `productSuggestions` query too? It is the query responsible for the autocomplete products